src/commands/distribute/lib/mc-fus-uploader/mc-fus-uploader.ts

```diff
@@ -0,0 +1,779 @@
+import {
```

Comment:
Should we put copyright here?

Reply:
If the original copyright/license requires it, then yes, please add it. Otherwise, it will be same copyright and license as the CLI as a whole.